code/game/machinery/vending.dm

```diff
@@ -3034,6 +3034,7 @@ var/global/num_vending_terminals = 1
 		/obj/item/key/security/spare = 1,
 		/obj/item/weapon/depocket_wand = 4,
 		/obj/item/weapon/ram_kit = 1,
+		/obj/item/borg/upgrade/cult = 2
```

Comment:
Tell me, why does this need to be here when it will appear to them as premium item with the other upgrades? I get you want to push your thing to be used but let's be consistent.

Reply:
I forgot about those upgrades. I doubt this will be used more than 1 in 10 matches, even if that. To make it only have a 1/5 or so chance of appearing each round would be a bit too much IMO.